### ISSUE TITLE
Principle 9: clarification on what constitutes a "usage"

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBOFoundry.github.io#2300]
There are some pretty inconsistent messages floating around here:

https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1008

and here:

https://obofoundry.org/principles/fp-009-users.html

According to https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1008 (this is what I was taught and required from everyone since I joined) a usage _must be evidenced_ by providing a link that reveals a specific ontology ID. 

I think we still don't have clarity here. @deepakunni3 can you make sure there is clear issue here that based on the 2 links above lists a few options for _operationalising the plurality of users_ issues? Mainly this is about _which evidence is sufficient?_.

A link to a paper? A mention of an ID? A link to a website claiming the ontology is used?

**Comment by matentzn** [comment]
When you are done preparing the issue, can you tag it with OFOC call?

**Comment by nataled** [comment]
The 'requirement' you mention that is given in #1008 is not a requirement, it is an example of usage as mentioned on that principle page. Remember, the principles represent the result of discussions and acceptance by the OFOC. The automated validations are, sometimes, approximations of those requirements. The principle itself is clear that evidence of usage must be given in the metadata file, and is clear as to what constitutes evidence. We would need to discuss this if either (1) you wish to expand on the ways to evidence usage, or (2) you wish to restrict. It sounds like you want the second.

**Comment by matentzn** [comment]
Yes, it seems I have not been paying attention to the verbatim of the principles text.. 

My issue is not what constitutes `usage` per se, I think this is ok in the current text.. But there are a few wholes as to what can count as evidence for usage. For example, there is no clarification if  when you import an ontology, all ontologies imported by that ontology automatically become used everywhere. For example, RO has some range restriction involving OGMS, everyone imports RO. Does it mean everyone is using OGMS? I hope not. Else all you need to do to get use by everyone in the world is to add a relation to RO with a domain that is in your own ontology. Also "A documentation page with links to databases using the ontology for annotation" is not that air tight, as I can  create a blog post saying "open targets is using my database" and fulfilling the verbatim (maybe not the spirit, but the verbatim). What we want is not a documentation page, but a link in open targets that resolves to a page that illustrates a specific term being used. 

In any case, we can treat this a request to formally tighten the condition of what constitutes usage _evidence_ to 

- Provide a link that resolve to a page that illustrates the use of a specific _term_ in the ontology. The page must either show the ID of the ontology term in the URL, or clearly mention the ID of the ontology term on the resolved page.

**Comment by nataled** [comment]
I'd need to track down the discussion of this from years ago, but I can say that your proposal was discussed in one or more OFOC calls and specifically rejected as too restrictive. 

IMO, this is the worst of the principles, as it has nothing to do with the quality of the ontology, the completeness/accuracy of it's metadata, or the interplay between developers and users. In fact, this past week I was contemplating that we should dispense with it altogether!

**Comment by matentzn** [comment]
:D I dont know about despense.. While I agree usage is probably not the most important (SNOMED is widely used but will fail many of the other principles), I think it is an important quality indicator for users: if you annotate your data with standard ontologies, you should get the most bang for buck, which is the ontology that is used most widely..

**Comment by addiehl** [comment]
Maybe the scope or definition of usage should be narrowed to term reuse in other ontologies and this should become an informational item only, not a requirement

**Comment by nataled** [comment]
> Provide a link that resolve to a page that illustrates the use of a specific term in the ontology. The page must either show the ID of the ontology term in the URL, or clearly mention the ID of the ontology term on the resolved page.

EWG discussion: Term use is indeed evidence of ontology usage (as indicated already in the Examples section), but it is only one type of evidence (others are listed). The instructions in the example already indicate that the evidence should be in the form of a link that shows the term usage. We note that the discussion here focuses on only one of the many examples of evidence of usage, and thus it cannot be made to be the sole mechanism to demonstrate usage. We will, instead, flesh out the requirements for ALL examples (work in progress).

**Comment by nataled** [comment]
See PR #2363
